doc/building-and-deploying-the-website.md

```diff
@@ -0,0 +1,22 @@
+## Building and Deploying (Automated)
+
+### Testing
+
+After you are finished developing and making your changes, make sure to test them. Run:
+
+	npm test
```

Comment:
You probably also want to run `gulp build` locally before committing, or have `npm test` run it for you.